base.js

```diff
@@ -15,6 +15,9 @@ module.exports = {
         // LWC lifecycle hooks validation
         '@lwc/lwc/no-deprecated': 'error',
 
+        // LWC public property syntax validation
+        '@lwc/lwc/no-leading-uppercase-api-name': 'error',
```

Comment:
Do we want to error out the existing ones? can we manually run it and have an estimate of how many fails this, and fix it before hand?